basic_ingest.install

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/**
+ * @file
+ * Update functions for the Basic Ingest module.
+ */
+
+/**
+ * Moves the help text to config.
+ */
+function basic_ingest_update_9001() {
+  $help_text = <<<EOHELP
+ <p>Adding this Repository Item to a collection isn't required, but can be used to organize items for display purposes. Use the 'Add container' button to facilitate multiple collection inheritance.</p>
+ <p>When creating a Repository Item using the 'Collection' content type, you will also be asked to optionally provide a 'Representative Image'; this will be used as a thumbnail, as well as in other situations where an image representative of the collection may be desirable.</p>
+ <p>By default, if any Media types are associated with the selected content type, you will be redirected to the Media ingest page after clicking 'Save'.</p>
+ <p>Uncheck 'Published' to keep this Repository Item out of public view. Unpublished items will still be preserved.</p>
+ EOHELP;
+
+  $current_help_text = \Drupal::config('node.type.islandora_object')
+    ->get('help') ?: '';
+  \Drupal::service('config.factory')->getEditable('node.type.islandora_object')
+    ->set('help', $current_help_text . PHP_EOL . $help_text)
```

Comment:
Probably fine, but this changes the order based on what I saw (reported in https://github.com/discoverygarden/basic_ingest/pull/26#discussion_r2325597481), where the value from the hook ended up _before_ the configured value (ultimately, might be up to the order of hook execution, which can always get weird?) .

Also, given that in the form this is a textual input, it looks like Windows line breaks should be used, with the full `\r\n` business, at least so indicated from looking at a dump of the YAML: 

```
[...]
help: "asdf test stuff\r\nmore words"
[...]
```